README.md

```diff
@@ -1636,6 +1637,9 @@ Default values depend on your operating system.
 
 **Note**: There is no official package available for mod\_proxy\_html and thus it must be made available by means outside of the control of the apache module.
 
+##### Class: `apache::mod::proxy_wstunnel`
+Includes the mod\_proxy_\wstunnel module on Apache versions >= 2.4.5. For versions below, no action is taken.
```

Comment:
The module doesn't exist below 2.4.5 (unless backported) but `::apache::version` doesn't offer finer resolution. I fixed the README to reflect the actual behavior. A relatively quick scan doesn't reveal any OS-packaged versions that are 2.4.4 or below.